tools/add_libconcept.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
This script needs bash; plain "sh" is not enough, because, it uses `x=$(blah)` syntax.

Reply:
Oh. I was using `dash`, hoping it is close enough to plain `sh`. But it seems only backticks would be supported. Fine. `bash` is everywhere we care about.